index.js

```diff
@@ -81,7 +81,13 @@ Rollout.prototype.get = function (key, id, opt_values, multi) {
         if (!modifiers[modName].condition) {
           modifiers[modName].condition = defaultCondition
         }
-        output = modifiers[modName].condition(opt_values[modName])
+        try {
+          output = modifiers[modName].condition(opt_values[modName])
+        }
+        catch (err) {
+          console.warn('rollout key[' + key + '] mod[' + modName + '] condition threw:', err)
```

Comment:
should we use a template literal here instead?

Reply:
`node-rollout` is an ES5 library at the moment, and template literals are an ES6 feature.